### PR TITLE
data: Trikcode

### PR DESCRIPTION
**Pull request by Trikcode**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2826"></a>

### CODE REVIEW

File: data/Trikcode.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "Nobert Patrick",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Full Stack Developer | **Open Source Contributor**",
+  "avatar": "https://github.com/Trikcode.png",
+  "tags": [
```
Comment (Njong392) [inline data/Trikcode.json:7]: These tags are okay, but the indentation seems really big.

File: data/Trikcode.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    },
+    {
+      "name": "GitHub: Let's collaborate",
+      "url": "https://github.com/Trikcode",
+      "icon": "FaGithub"
+    },
+				{
+      "name": "LeetCode: Check Me out",
+      "url": "https://leetcode.com/NobertPatrick/",
+						"icon": "FaLeetcode"
```
Comment (Njong392) [inline data/Trikcode.json:39]: Same goes for the indentation of `icon` here

File: data/Trikcode.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    },
+    {
+      "name": "LinkedIn: Let's connect",
+      "url": "https://www.linkedin.com/in/nobert-patrick-a3001896/",
+      "icon": "FaLinkedin"
+    }
+			]
```
Comment (Njong392) [inline data/Trikcode.json:46]: And then there's this single parentheses, `]` that's too far in as well.

**Review by Njong392 — CHANGES_REQUESTED** [review]
Everything looks great! Thank you for contributing :tada: Please check out the little reviews I've made.

File: data/Trikcode.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Full Stack Developer | **Open Source Contributor**",
+  "avatar": "https://github.com/Trikcode.png",
+  "tags": [
+			"Python",
+			".NET/C#",
+			"Javascript",			
+			"Typescript",
+			"Open Source"
```
Comment (Njong392) [inline data/Trikcode.json:12]: These tags look great, but there seems to be too much indentation here. Shifting them out a little will look great.
```suggestion
  "tags": [
        "Python",
	".NET/C#",
	"Javascript",			
	"Typescript",
	"Open Source"
```

**Review by Njong392 — CHANGES_REQUESTED** [review]

File: data/Trikcode.json (hunk 4 above)
Comment (Trikcode) [reply data/Trikcode.json]: Thanks so much I have updated the file

File: data/Trikcode.json (hunk 2 above)
Comment (eddiejaoude) [reply data/Trikcode.json]: We have a GitHub Action that will fix the formatting after merge as long as the json is valid - we have a GitHub Action that also check the json is valid in the pull request before merging

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍  Great collaboration everyone 🎉

**Pull request merged**